awx/playbooks/action_plugins/playbook_integrity.py

```diff
@@ -88,7 +87,7 @@ def validate_checksum_manifest(self):
         if not os.path.exists(manifest_file):
             return {
                 "failed": True,
-                "msg": f"Expected file not found: {path}",
+                "msg": f"Expected file not found: {manifest_file}",
```

Comment:
@relrod the `path` variable is referenced in other places in the file, but I don't think it's in scope here.

Reply:
Yep, copypaste fail. Good catch!

Reply:
ok, I think that this PR won't make it in the current development sprint. If anything looks urgent enough that it needs fixing before we ship, you should consider making a minor PR that just makes those changes without the linting rules.